src/shared/ui/video-player/video-player.module.scss

```
@@ -0,0 +1,15 @@
+.youtube-player-container {
+  position: relative;
+  overflow: hidden;
+  height: 0;
+  padding-bottom: 56.25%;
+}
+
+.youtube-player-container iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+
```

let's add border radius as the design shows

```suggestion
  border-radius: 12px;
```

---

added